### ISSUE TITLE
How to reset zoom in/out ?

### ISSUE THREAD

**Issue by mhdramadhanarvin** [SanichKotikov/pinch-zoom-pan#12]
Is this repository still being maintained? I want to ask how to reset zoom in/out ?

**Comment by SanichKotikov** [comment]
@mhdramadhanarvin hi. Right now, there is no way to do this, but I think I can add it in few next days.

**Comment by SanichKotikov** [comment]
@mhdramadhanarvin published `v4.0.0`

```js
const canvas = create(...);

canvas.update({ z: 1 });
// or reset both, position and zoom
canvas.reset();
```
